viewsgetter.bat

```diff
@@ -1,20 +1,20 @@
-REM username
+@rem username
```

Comment:
Um i dont think that this is how it works for windows...REM has been working so idk

Reply:
aight ig

Reply:
done